README.md

```diff
@@ -84,6 +84,17 @@ Delete cookie:
 Cookies.remove('name');
 ```
 
+Please notice that removing unexisting cookie does not raise any exception nor return any value
+
+```javascript
+var whatWasTheValue = Cookies.remove('nothing');
+if (whatWasTheValue === undefined) {
+  console.log("I will be excuted");
+} else {
+  console.log("I will never be executed");
```

Comment:
I understand. What about something like this then:

```
Delete a cookie that is visible on the current page:

Cookies.set('name', 'value', { path: '' });
Cookies.remove('name'); // does not remove the cookie
Cookies.remove('name', { path: '' }); // removes the cookie
```

We can then consistently use "visible" instead of "valid" where we are talking about the cookie visibility based on the attributes.

Unfortunately, we can't document that `Cookies.remove()` returns `undefined` because that's not a supported behavior. It is actually `void` and returns nothing.

Make sense?